build/assets/configs/99-runtimes.conf

```diff
@@ -0,0 +1,10 @@
+# We should copy paste the default runtime because this snippet will override the whole runtimes section
+[crio.runtime.runtimes.runc]
+runtime_path = ""
+runtime_type = "oci"
+runtime_root = "/run/runc"
+
+[crio.runtime.runtimes.high-performance]
```

Comment:
do we need to update the commit message?
Could you please document (or link the documentation) about how the crio configuration works now? before using the wrapper was pretty easy to get, but now it feels like something's missing

Reply:
Questions to clarify why is not very clear (!!!) - if the functionality is baked in crio, and if we just need to provide a new separate runtime class (if so this deserve a comment), why the two runtime classes aren't 1:1 clones?
IOW, why for the default runtime class runtime_path = ""?

Reply:
Btw, is the name fixed? Low latency is specifically NOT high performance. Quite the opposite.

Reply:
done

Reply:
@MarSik Yes, it hard-coded under the CRI-O, I was pretty under the expression that providing the low latency will give you a better performance at least for the process that will run under the container configured with it. Like we named our operator as performance-addon-operator.

Reply:
ok, good enough for me now